doc/examples/segmentation/plot_regionprops_table.py

```diff
@@ -0,0 +1,46 @@
+"""
+=========================
+Tabulate areas of regions
```

Comment:
Maybe a more generic title such as "Explore and visualize region properties with pandas"?

Reply:
I'm okay generalizing to (all possible) region properties. With "tabulate" I meant to express "pandas" ;)
The only think about "Explore and visualize ... with pandas" is that I would expect the example not to depend on seaborn, but to use the matplotlib API via pandas:
```py
areas.plot(x='volume fraction', y='area', kind='scatter')
```
which results in a less fancy plot, but saves three lines of code, and looks more readable / powerful.

Reply:
I agree with self-contained, and maybe readable, but certainly not more powerful! =)

Reply:
@jni Sure, seaborn is much more powerful in absolute terms! What I had in mind was that we get to showcase the power of pandas' plotting backend *in addition to* (hence, more) its 'base' capabilities of manipulating tables.
In any case, I think we want to either reflect the example title ("Explore and visualize region properties with pandas") or change the title to reflect the example.